package aa.exception.handling_rules;

public class Test02 {
	
	static void m1() {
		System.out.println("1");
		try {
			System.out.println("2");
			System.out.println("3");
			System.out.println("4");
		}catch(Exception e) {
			System.out.println("5");
			System.out.println("6");
			System.out.println("7");
		} System.out.println("8");
	}
	public static void main(String [] args) {
		
		System.out.println("main start");
		m1();
		System.out.println("min end");		
	}

}
